src/forDefi/forDefi.ts

```diff
@@ -57,7 +57,7 @@ export async function sendToForDefi(unsignedTxidFile: string, ctxFile: string, w
     const timestamp = new Date().getTime();
     const payload = `${path}|${timestamp}|${requestBody}`;
 
-    const privateKeyFile = "../private.pem"
+    const privateKeyFile = "./private.pem"
```

Comment:
not sure but is it correct to store the private key in cli?

Reply:
This change was requested by the flare team. they wanted us to change the directory from the parent dir to the current dir for both token and pem file.